src/cmd/cli/command/commands.go

```diff
@@ -293,7 +293,7 @@ var rootCmd = &cobra.Command{
 		}
 
 		if err != nil {
-			cli.Debug(" - Could not load docker compose file: ", err)
```

Comment:
I don't agree with this one: there's a bunch of CLI commands that don't need compose files, and they'd show a warning. Like `login`.

Reply:
Good point, the goal was to surface any compose file parsing error, which, if not reported here, would be lost down the stream during he compose commands. I changed it to only report the warning if it is not `ErrNotFound`, what do you think?

Reply:
Or do something like the `authNeeded` annotation: add `projectNeeded` to the cmds that need it